Logic 1.0

```diff
@@ -1,10 +1,10 @@
 #Logic for the ideation stage
 
 1. Start
-2. Invite digital twin of note taker to online meeting
-3. Participate in online meeting like other members
-4. Record the minutes
-5. Prepare notes as verbatim
-6. Save the minutes and notes in a drive
+2. Enable twin note taker as a bot type software
+3. As the option of caption/subtitle this will record all captions and Audio and video from each user as this give a detailed note of who spoke with time and duration of speach.
+4. If there is any special notes to be taken more importantly we can use a special assist called upon a keyword. 
+5. All the data will be saved in the drive with audio and video clips for future uses. And a detailed excel sheet with detailed link attached with each users name for easy acess. 
+6. Give a short summary of the meeting and send an email to all participants 
```

Comment:
Saving and sending summary as emails will be included as a feature for Digital Twin Note Taker